client/my-sites/checkout/checkout/checkout.jsx

```diff
@@ -264,6 +267,7 @@ const Checkout = createReactClass( {
 	getCheckoutCompleteRedirectPath() {
 		let renewalItem;
 		const { cart, selectedSiteSlug, transaction: { step: { data: receipt } } } = this.props;
+		const domainReceiptId = get( cartItems.getGoogleApps( cart ), '0.extra.receipt_for_domain', 0 );
```

Comment:
😍 for using `get`, one of my favourite functions from lodash. I didn't know `0.extra` was an accepted format, I've always used `[0].extra`. The `[0]` notation might easier to understand for people not familiar with this "magic" method, but that's up to you.